s3-commons/src/test/java/io/aiven/kafka/connect/iam/AwsCredentialProviderFactoryTest.java

```diff
@@ -32,49 +32,50 @@
 import org.junit.jupiter.api.Test;
 
 final class AwsCredentialProviderFactoryTest {
-
     private AwsCredentialProviderFactory factory;
     private Map<String, String> props;
 
     @BeforeEach
     public void setUp() {
         factory = new AwsCredentialProviderFactory();
         props = new HashMap<>();
-        props.put(S3SinkConfig.AWS_S3_BUCKET_NAME_CONFIG, "anyBucket");
-        props.put(S3SinkConfig.AWS_S3_BUCKET_NAME_CONFIG, "any-bucket");
+        props.put(S3SinkBaseConfig.AWS_S3_BUCKET_NAME_CONFIG, "any-bucket");
```

Comment:
Like mentioned above, we can make this generic test depend on common config, instead of sink.